allennlp/data/token_indexers/single_id_token_indexer.py

```diff
@@ -34,6 +34,7 @@ def __init__(
         lowercase_tokens: bool = False,
         start_tokens: List[str] = None,
         end_tokens: List[str] = None,
+        feature_name: str = "text",
```

Comment:
Hmm, I think this would actually let us entirely remove the `Ner` and `PosTag` indexers.  Probably should have done this a long time ago...

Can you add a note to the docstring about this new parameter?